lib/manageiq/loggers/container.rb

```diff
@@ -64,11 +64,12 @@ def translate_error(level)
         end
 
         def request_id
-          if Thread.current[:current_request]&.request_id
-            ActiveSupport::Deprecation.warn("Usage of `Thread.current[:current_request]&.request_id` will be deprecated in version 0.5.0. Please switch to `Thread.current[:request_id]` to log request_id automatically.")
+          Thread.current[:request_id] || Thread.current[:current_request]&.request_id.tap do |request_id|
```

Comment:
is the `tap` on the second part or the whole thing?
if the latter, would it read better as
```ruby
(Thread.current[:request_id] || Thread.current[:current_request]&.request_id).tap do |request_id|
```

Reply:
Second part only